src/M6Web/Bundle/LogBridgeBundle/Config/FilterParser.php

```diff
@@ -59,11 +67,11 @@ protected function isRoute(string $name): bool
     public function parse(string $name, array $config): Filter
     {
         if (
-            !array_key_exists('route', $config) ||
+            (!array_key_exists('route', $config) && !array_key_exists('routes', $config)) ||
```

Comment:
By doing this check, you only validate that route or routes are configured, but what happens if we configure route and routes ? Do we want to allow both configuration ? And by looking line 85 we can see that only one configuration can be used. If route → route, if routes → routes, if route and routes → route.
I think that we need a logical XOR check.